### DISCUSSION TITLE
Importing a CSV file into Yeti Force

### DISCUSSION THREAD

**Discussion by Jkhibeir** [YetiForceCompany/YetiForceCRM#16354] [How to]
I have uploaded a CSV File several times to YetiForce. It will state that the components have uploaded successfully, but can not find the file thereafter. Does anybody know where you can find your uploaded file? Thanks!

**Comment by rskrzypczak** [comment]
Please provide a more detailed description, because I don't know what you are asking.

**Comment by Jkhibeir** [comment]
Hello, 

 

Thank you for responding. I exported a csv file that contains all my projects, and when I went to import into YetiForce, it didn’t display my uploaded CSV file onto YetiForce. I am not sure if that helps you, but happy to jump on a zoom call to explain further. Thanks again!
